File: problems/s/sorttwonumbers/sorttwonumbers.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int a, b;
    if (scanf("%d %d", &a, &b) != 2) {
        printf("Failed to read a and b!");
    }

    if (a > b) {
        printf("%d %d", b, a);
    }
    else {
        printf("%d %d", a, b);
    }

    return 0;
}

File: problems/s/sorttwonumbers/sorttwonumbers.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    a, b = [int(val) for val in input().split()]
    if a > b:
        print(f'{b} {a}')
    else:
        print(f'{a} {b}')


if __name__ == '__main__':
    main()